Added Rhoda Michael profile on guides team

sakibian opened the pull request:
Fixes #5623 

### What changes did you make?
  - Added Rhoda Michael's profile on the guides team board

### Why did you make the changes (we will use this info to test)?
  - Information was not there, added a new name to the board

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

![image](https://github.com/hackforla/website/assets/40847839/c9a01b55-b8b6-44e1-9d61-76bbb9b9eea0)

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
![image](https://github.com/hackforla/website/assets/40847839/23e6475d-46ca-4856-a4cb-56d7df866b50)

</details>

sakibian committed:
added Rhoda Michael profile on guides team

Signed-off-by: sakibian <sakib192@gmail.com>

KyleA99 commented:
Review ETA: 6 PM 10/10/23
Availability: 5-8 PM Tuesday

KyleA99 reviewed (dismissed):
Reviewed and approved.  Looks great @sakibian 

I am curious, is your "visuals before changes applied" image from the actual deployed website?  I'm assuming it is because it shows Sarah Levinen's profile, but the codebase does not have her profile info.  Might be a good practice to have your "visuals before changes applied" image taken from an initial docker build prior to any file changes, rather than taking screenshot from deployed site.  Not sure if this matters, just a thought.

Thinking-Panda commented:
Availability: Weekdays
ETA: 10/11

sakibian commented:
> Reviewed and approved. Looks great @sakibian
> 
> I am curious, is your "visuals before changes applied" image from the actual deployed website? I'm assuming it is because it shows Sarah Levinen's profile, but the codebase does not have her profile info. Might be a good practice to have your "visuals before changes applied" image taken from an initial docker build prior to any file changes, rather than taking screenshot from deployed site. Not sure if this matters, just a thought.

Yes @KyleA99, it is from the deployed site and when I try to `pull upstream` I don't see Sarah Levinen's profile in the code base. I thought I would get an updated code after using `pull upstream`.

KyleA99 commented:
@sakibian neither do I, I observed the same thing

sakibian commented:
That's interesting 🤨🤔

Thinking-Panda reviewed (changes requested):
@sakibian , I had the same situation. Please check if your remote codebase has "Sarah Levine" listed. Then, try `git pull --rebase` on your `gh-pages` branch to make it up-to-date

sakibian committed:
Update guides team with missing profile

Signed-off-by: sakibian <sakib192@gmail.com>

sakibian commented:
@Thinking-Panda is it good this time?
![image](https://github.com/hackforla/website/assets/40847839/182d5c0a-c432-4f98-b915-acc6f20dde13)

KyleA99 commented:
@sakibian looks good to me.  I guess we were both pulling in git incorrectly

sakibian commented:
@KyleA99 I think we weren’t wrong, we were using correct commands but for some reason our gh-pages wasn’t updating, may be updated code was missing, but after two days it gets fixed, May be Levin's code contributer had merge conflict or something we unaware of.

Thinking-Panda reviewed (commented):
@sakibian , Just checking, did you type in "Sara Levine" details or you pulled it from the published website codebase?

Thinking-Panda commented:
> @KyleA99 I think we weren’t wrong, we were using correct commands but for some reason our gh-pages wasn’t updating, may be updated code was missing, but after two days it gets fixed, May be Levin's code contributer had merge conflict or something we unaware of.

Levine's details were merged recently and that why this pull request has merge conflicts to be resolved. Resolving that will clear the PR to be approved. Helpful resource- https://github.com/hackforla/website/wiki/How-to-Resolve-Merge-Conflicts

sakibian commented:
> @sakibian , Just checking, did you type in "Sara Levine" details or you pulled it from the published website codebase?

After review changes requested I did `git pull upstream gh-pages` saw the updated `guides-team.md` file added to my local. From here I did check out to `sakibian:update-guides-team-profile-5623` branch, where my previous part of the code was available and `pushed` and `pull requested` were created here, and then I just manually copy paste Sara Levin's
profile to my current branch and updated the code. [1. Updated Levin's profile here](https://github.com/hackforla/website/commit/6639b962cca2a8e3740ff30bbeb05e3c6d4e7b99 ) [2. Previous Added Rhoda Michael profile on guides team](https://github.com/hackforla/website/pull/5692/commits/7423e2c7ee87ae294606a1c31e2c2c08480b253e)

sakibian commented:
@Thinking-Panda another way around with updated code, check this: [5733](https://github.com/hackforla/website/pull/5733)

I'll close one pull depending on workarounds.
